pkg/state/types.go

```diff
@@ -1,42 +1,19 @@
 package state
 
+// DB interface for database
 type DB interface {
 	// Load should retrieve local state
 	Load() error
 	// GetState
-	GetState() StateI
+	GetState() State
 	// Save should save networks data to local state
 	Save() error
 	// Delete should delete networks state
 	Delete() error
 }
 
+// StateI interface for state
 type StateI interface {
```

Comment:
Don't think we will need this after we have a StateGetter interface because it will return State concrete type that has the NetworkMap